### ISSUE TITLE
Shared cache sqlite db files not removed

### ISSUE THREAD

**Issue by JeffPsycle** [rianhunter/dbxfs#48]
Hello,

After using DBXFS for years now I just discovered an issue that the SQLite DB files created in `/tmp` never get cleared out. This hasn't been an issue until recently as there was a long period between kernel patches on our machines. For example in the last 4-5 days we have 50k+ files floating about in `/tmp`. 

```bash
[root@ip monitoring]# find /tmp -name *cache=shared* | wc -l
53533
```

I would just routinely flush `/tmp` but we have found DBXFS is less than happy to have open cache db files yanked out while in use. Some of the files in our use case are still in use after more than 24 hours, so picking an arbitrary time with a find/delete isn't really a good option. So I have been removing anything not coming up from `lsof`.

```bash
[root@ip monitoring]# lsof -w | grep 'dbxfs' | grep '/tmp/tmp' | awk '{print $11}' | sort | uniq

/tmp/tmp0_61wpux&cache=shared
/tmp/tmp7mwhh67g&cache=shared
/tmp/tmpku2_bpgs&cache=shared
/tmp/tmpm_3hu2pb&cache=shared
/tmp/tmpp3s0fj_p&cache=shared
/tmp/tmpqtxcehs2&cache=shared
```

I think this might be fixed with an unlink of the db file here:

https://github.com/rianhunter/dbxfs/blob/60e206e3b7ae3e2fabb23f7af168a21a925f22dc/dbxfs/cachingfs.py#L671-L680

`self._file_path` gets removed, but `self._db_file` does not. 

Thanks for your time on reviewing this issue.

**Comment by rianhunter** [comment]
This is a duplicate of #39 but the fix you suggest would not work. Currently do not have the time to review this issue so you're on your own. If this is a high priority issue I can be contracted to fix this issue, though my time is expensive.

**Comment by rianhunter** [comment]
Fixed in  ba302ed
